c/include/libsbp/tracking.h

```diff
@@ -79,9 +79,9 @@ signal is in continuous track.
  * measured signal power.
  */
 typedef struct __attribute__((packed)) {
-  u8 state;    /**< Status of tracking channel */
-  sbp_gnss_signal_t sid;      /**< GNSS signal being tracked */
-  float cn0;      /**< Carrier-to-noise density [dB Hz] */
+  gnss_signal16_t sid;    /**< GNSS signal being tracked */
+  u8 fcn;    /**< Frequency channel number */
+  u8 cn0;    /**< Carrier-to-Noise density.  Zero implies invalid cn0. [dB Hz / 4] */
```

Comment:
yes but the "final final" decision was to be consistent with what had been decided for MSG_OBS, thus 0 being invalid!